jekyll/_layouts/classic-docs.html

```diff
@@ -27,20 +27,33 @@
             <div>
               {% if page.page-type != "index" and page.page-type != "homepage" %}<h1 id="{{ page.title | downcase | split: " " | join: "-" }}">{{ page.title }}</h1>{% endif %}
             </div>
-            <div id="tooltip" role="tooltip" class="tooltip-popover">
-              This document is applicable to the following:
-              <div id="arrow" data-popper-arrow></div>
+
+            <!-- Quick info list - time posted + server version -->
+            {% if page.page-type != "index" and page.page-type != "homepage" %}
+            <div id="tooltip" role="tooltip" class="tooltip-popover"></div>
+            <div id="tooltip-time" role="tooltip" class="tooltip-popover">
+              Last updated
+              <div id='arrow' data-popper-arrow></div>
             </div>
-            {% if page.version %}
-            <div class="server-version-wrapper">
-              {% for version in page.version %}
-              <span class="badge badge-primary">
-                <button class="server-version-badge" aria-describedby="tooltip">{{ version }}</button>
+
+            <div class="quick-info-list">
+              <time aria-describedby="tooltip-time" id="time-posted-on" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
```

Comment:
I like how semantic this is. this is a good opportunity to add sn itemprop SEO schema tag as well:
https://schema.org/datePublished

Let's reach out to SEO to see what they think.

more on how to use schema: https://schema.org/docs/gs.html#microdata_itemscope_itemtype

Reply:
Will added to DD-266